modules/phase_field/include/auxkernels/SolutionAuxMisoBnds.h

```diff
@@ -0,0 +1,57 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "AuxKernel.h"
+
+class SolutionUserObject;
+
+/**
+ * AuxKernel for reading a solution from file.
+ * Creates a function that extracts values from a solution read from a file,
+ * via a SolutionUserObject. It is possible to scale and add a constant to the
+ * solution read.
+ */
+class SolutionAuxMisoBnds : public AuxKernel
```

Comment:
This is largely a copy paste of `SolutionAux`. You should inherit from that class instead. Also please do not use abbreviations like "Miso" for Misorientation. Just spell it out (same with Bnds -> Boundaries).

Reply:
Thank you! I just updated them. Just let me know if you have any suggestions.